psql2mysql.py

```diff
@@ -317,6 +321,29 @@ def do_migration(config):
         sys.exit(1)
 
 
+# restrict the source database to postgresql for now
+def check_source_schema(source):
+    if uri_reference(source).scheme != "postgresql":
+        print('Error: Only "postgresql" is supported as the source database '
+              'currently', file=sys.stderr)
```

Comment:
I sort of feel like we should be setting up a logger here, which will get us to sys.stderr as well

Reply:
I think @rhafer removed previous logger usage for some reason

Reply:
@rhafer Why did you remove the logger? Sounds like something we want rather than relying on customers having to copy and paste terminal output?